src/test/record/test_frequency.py

```diff
@@ -0,0 +1,389 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from caret_analyze.record import ColumnValue
+from caret_analyze.record.frequency import Frequency
+from caret_analyze.record.record_factory import RecordFactory, RecordsFactory
+
+
+def create_records(records_raw, columns):
+    records = RecordsFactory.create_instance()
+    for column in columns:
+        records.append_column(column, [])
+
+    for record_raw in records_raw:
+        record = RecordFactory.create_instance(record_raw)
+        records.append(record)
+    return records
+
+
+def to_dict(records):
+    return [record.data for record in records]
+
+
+class TestFrequencyRecords:
+
+    def test_callback_empty_case(self):
+        records_raw = [
+        ]
+        columns = [ColumnValue('/node/callback/callback_start_timestamp'),
+                   ColumnValue('/node/callback/callback_end_timestamp')]
+        records = create_records(records_raw, columns)
+
+        callback_frequency = Frequency(records, target='callback')
+
+        expect_raw = [
+        ]
+        result = to_dict(callback_frequency.to_records(
+            initial_timestamp=1000000000))
+        assert result == expect_raw
+
+    def test_callback_normal_case(self):
+        records_raw = [
+            {'/node/callback/callback_start_timestamp': 1000000000,
+             '/node/callback/callback_end_timestamp': 1000001000},
+            {'/node/callback/callback_start_timestamp': 1000500000,
+             '/node/callback/callback_end_timestamp': 1000501000},
+            {'/node/callback/callback_start_timestamp': 2000000000,
+             '/node/callback/callback_end_timestamp': 2000001000},
+            {'/node/callback/callback_start_timestamp': 2000300000,
+             '/node/callback/callback_end_timestamp': 2000301000},
+            {'/node/callback/callback_start_timestamp': 2000500000,
+             '/node/callback/callback_end_timestamp': 2000501000},
+        ]
+        columns = [ColumnValue('/node/callback/callback_start_timestamp'),
+                   ColumnValue('/node/callback/callback_end_timestamp')]
+        records = create_records(records_raw, columns)
+
+        callback_frequency = Frequency(records, target='callback')
```

Comment:
ここは以下でお願いします。

```suggestion
        callback_frequency = Frequency(records)
```

インターフェースとしては以下のようにおねがいします
```
def __init__(records*, target_column: Optional[str] = None):
    pass
```
カラム名と完全一致で、デフォルトとしては一番左のカラム名を使うようにしてください。